### PR TITLE
iOS url launcher to use SafariServices instead of changing to Safari app

### PR DESCRIPTION
**Pull request by xster**
https://github.com/flutter/flutter/issues/11584

(drive-by update to cocoapods for example)

### CODE REVIEW

**Comment by tvolkert** [comment]
FYI @harveynick @mehmetf

File: packages/url_launcher/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: url_launcher
 
-version: 0.4.2+5
+version: 0.4.3
```
Comment (tvolkert) [inline packages/url_launcher/pubspec.yaml]: Let's bump the version to 1.0.0 (per a previous discussion, our plugins don't appear to be "launched" if their versions are <1.0.0)

File: packages/url_launcher/example/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -257,9 +257,14 @@
 			files = (
 			);
 			inputPaths = (
+				"${SRCROOT}/Pods/Target Support Files/Pods-Runner/Pods-Runner-frameworks.sh",
+				"${PODS_ROOT}/../../../../../../flutter/bin/cache/artifacts/engine/ios/Flutter.framework",
```
Comment (tvolkert) [inline packages/url_launcher/example/ios/Runner.xcodeproj/project.pbxproj]: Is this relative path stable across different environments?  Should we have a `${FLUTTER_ROOT}` variable instead?

Comment (xster) [reply packages/url_launcher/example/ios/Runner.xcodeproj/project.pbxproj]: Seems like cocoapods is continuously resetting this value after I change it manually. Just reverted this file.

File: packages/url_launcher/pubspec.yaml (hunk 1 above)
Comment (xster) [reply packages/url_launcher/pubspec.yaml]: Ah ok, done

**Pull request merged**